Core/Core/Login/LoginWebViewController.swift

```diff
@@ -96,7 +96,9 @@ public class LoginWebViewController: UIViewController, ErrorViewController {
         setupIndeterminateLoadingIndicator()
 
         // Manual OAuth provided mobileVerifyModel
-        if mobileVerifyModel != nil {
+        if let mobileVerifyModel {
+            // Modify the title to include the url scheme to easily catch http/https errors.
+            title = mobileVerifyModel.base_url?.absoluteString
```

Comment:
Is it possible that the `base_url` is actually `nil`?
If so, then this would clear out the title.
Would it make sens to wrap this title assignment (but not the `return`) in a `if let modelTitle = ...` or similar?